bitnami/scylladb/templates/service.yaml

```diff
@@ -0,0 +1,75 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    app.kubernetes.io/part-of: scylladb
+    app.kubernetes.io/component: scylladb
+    {{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+    {{- /* Adding extra selector for the ServiceMonitor object to avoid duplicate targets  */}}
+    servicemonitor-scrape: true
+    {{- end }}
```

Comment:
Is this a standard label? I think it's the 1st time we add it to the catalog.

Reply:
I don't think it is, but as we have several services with the same targets, We added this label to avoid duplicate targets. If there's a better alternative that does not collide with other prometheus labels, I'll be glad to add it.